boring/Cargo.toml

```diff
@@ -16,20 +16,38 @@ features = ["rpk", "pq-experimental"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-# Use a FIPS-validated version of boringssl.
+# Use the FIPS-validated version of boringSSL. This feature is incompatible
+# with any feature that requires (patches to) the latest version of boringSSL.
 fips = ["boring-sys/fips"]
 
-# Link with precompiled FIPS-validated `bcm.o` module.
-fips-link-precompiled = ["fips", "boring-sys/fips-link-precompiled"]
+# Link with a precompiled FIPS-validated `bcm.o` module in order to use the
+# FIPS-validated implementations of key exchange and encryption algorithms.
+fips-link-precompiled = ["boring-sys/fips-link-precompiled"]
 
 # Enables Raw public key API (https://datatracker.ietf.org/doc/html/rfc7250)
 rpk = ["boring-sys/rpk"]
 
 # Enables experimental post-quantum crypto (https://blog.cloudflare.com/post-quantum-for-all/)
 pq-experimental = ["boring-sys/pq-experimental"]
 
+# Support PQ key exchange by default. The client will prefer classical key
+# exchange, but will upgrade to PQ key exchange if requested by the server.
+# This is the safest option if you don't know if the peer supports PQ key
+# exchange.
+pq-supported = ["pq-experimental"]
+
+# Prefer PQ key exchange by default. The client will prefer PQ exchange, but
+# fallback to classical key exchange if requested by the server. This is the
+# best option if you know the peer supports PQ key exchange.
+pq-preferred = ["pq-experimental", "pq-supported"]
```

Comment:
`pq-preferred` is confusing as one might think it's a preferred configuration, rather than PQ exchange being preferred, let's try to reflect the meaning in feature names

Reply:
What about "pq-key-exchange-preferred"?

Reply:
Changed to "pq-kx-preferred".